Sources/StreamChat/Workers/CurrentUserUpdater.swift

```diff
@@ -202,6 +227,14 @@ extension CurrentUserUpdater {
         }
     }
     
+    func loadBlockedUsers() async throws -> [BlockedUserDetails] {
+        try await withCheckedThrowingContinuation { continuation in
+            loadBlockedUsers { error in
```

Comment:
Naming: it is actually `result`, not `error` here.

Reply:
Updated 👍